vm/vm.go

```diff
@@ -212,6 +212,7 @@ func (vm *VM) initConstants() {
 		vm.initNullClass(),
 		vm.initArrayClass(),
 		vm.initHashClass(),
+		vm.initConcurrentHashClass(),
```

Comment:
I think it should be a standard library that need to be required when needed. If so, you should put the initialize method here:

```go
var standardLibraries = map[string]func(*VM){
	"net/http":          initHTTPClass,
	"net/simple_server": initSimpleServerClass,
	"uri":               initURIClass,
	"db":                initDBClass,
	"plugin":            initPluginClass,
	"json":              initJSONClass,
        // Here
        "concurrent_hash": vm.initConcurrentHashClass,
}
```

Note that the builtin classes and standard library class's initialize method is slightly different.